pkg/cache/v3/linear.go

```diff
@@ -592,8 +627,27 @@ func (cache *LinearCache) CreateWatch(request *Request, sub Subscription, value
 		cache.log.Warnf("[linear cache] error computing watch response: %s", err)
 		return nil, fmt.Errorf("failed to compute the watch respnse: %w", err)
 	}
-
+	shouldReply := false
 	if response != nil {
+		// If the request
+		//  - is the first
+		//  - provides a non-empty version, matching the version prefix
```

Comment:
why is there a condition for the version prefixes here? (I don't even see it in the code, so perhaps it's just the comment?)
I think the concept shouldn't even exist when stable version are used, since the whole point is to reuse versions across reconnects (presumably also across processes and restarts).

Reply:
The prefix is actually covered through the check of `ignoreCurrentSubscriptionResources`. I agree VersionPrefix makes little sense in this case, but I'm not sure we want to fully reject then
Tbh removing version prefix would allow to simplify quite a lot of code, and if we find no major issues with stable versions I might push for it

Reply:
Yes. This potential cleanup and the fact that version prefix does not serve a real purpose with stable versions may be worth a comment, for future readers.